lua/neo-tree/sources/buffers/commands.lua

```diff
@@ -52,6 +52,14 @@ M.copy = function(state)
   cc.copy(state, redraw)
 end
 
+M.copy_path = function(state)
```

Comment:
I don't think this is needed at all because there is a
```
cc._add_common_commands(M)
```
call at the end of the file, which will add this command anyhow. This pattern of wrapping common commands is only needed when we need to alter the common command in some way, such as by adding a `redraw` callback.